Chessington.GameEngine/Pieces/Queen.cs

```diff
@@ -10,7 +10,40 @@ public Queen(Player player)
 
         public override IEnumerable<Square> GetAvailableMoves(Board board)
```

Comment:
Queen is a just a combination of moves of Rook and Bishop, isn't it?

maybe you can refactor the code and make this methods reusable?